src/semgrep_agent/formatter.py

```diff
@@ -68,9 +84,69 @@ def _print_error_message(violation: Finding) -> str:
     return f"{BOLD}{rule}{END}"
 
 
-def dump(findings: Set[Finding]) -> None:
+OSC_8 = "\x1b]8;;"
+BEL = "\x07"
+
+
+def is_child_process_of(pattern: Pattern) -> bool:
+    """
+    Returns true iff this process is a child process of a process whose name matches pattern
+    """
+    me = psutil.Process()
+    parents = me.parents()
+    matches = iter(0 for p in parents if pattern.search(p.name()))
+    return next(matches, None) is not None
+
+
+LINK_PRINTER_PATTERN = re.compile("(iterm2|gnome-terminal)", re.IGNORECASE)
+LINK_WIDTH = 2 * len(OSC_8) + 2 * len(BEL)
+DO_PRINT_LINKS = is_child_process_of(LINK_PRINTER_PATTERN)
+
+
+def render_link(
+    text: str,
+    href: Optional[str],
+    print_alternative: bool = True,
+    width: Optional[int] = None,
+    pipe: TextIO = sys.stdout,
+) -> str:
+    """
+    Prints a clickable hyperlink output if in a tty; otherwise just prints a text link
+    :param text: The link anchor text
+    :param href: The href, if exists
+    :param print_alternative: If true, only emits link if OSC8 links are supported, otherwise prints href after text
+    :param width: Minimum link width
+    :param pipe: The text IO via which this link will be emitted
+    :return: The rendered link
+    """
+    is_rendered = False
+    if href:  # Don't render if href is None or empty
+        if pipe.isatty() and DO_PRINT_LINKS:
+            text = f"{OSC_8}{href}{BEL}{text}{OSC_8}{BEL}"
+            is_rendered = True
+            if width:
+                width += LINK_WIDTH + len(href)
+        elif print_alternative:
+            text = f"{text} {href}"
+
+    if width:
+        text = text.ljust(width)
+
+    # Coloring has to occur after justification
+    if is_rendered:
+        text = style(text, fg=Colors.LINK)
+
+    return text
+
+
+def build_action_url(v: Finding, c: Sapp, meta: GitMeta) -> str:
+    projectName = meta.repo_remote_origin
```

Comment:
```suggestion
    projectName = meta.repo_name
```

Isn't this the correct name if we want this to identify the configurable project?